tools/debugging/extract_sp_exceptions.sh

```diff
@@ -42,94 +40,81 @@ else
 fi
 
 DESTINATION_DIR="$(realpath -s "${DESTINATION_DIR}")/$(date +%m-%d-%Y)"
-mkdir -p "$DESTINATION_DIR"
-
-function download_pfs_logs {
-    container=$1
-    ssh root@services-dev.raiden.network "cd raiden-services/deployment/; docker-compose logs ${container} | gzip" > "${DESTINATION_DIR}/${container}.log.gz"
-    if [[ $? -ne 0 ]]; then
-        echo "Error: failed to download pfs-goerli logs"
-        exit 1
-    fi
-}
-
-function download_nodes_logs {
-    scenario=$1
-    current_server=$2
-    run_number=$3
-
-    nodes=$(${CURL_COMMAND} "${current_server}${scenario}" | grep '^<a ' | cut -d\" -f2 | grep "^node_${run_number}")
+SCENARIOS_DIR="$DESTINATION_DIR"/scenarios
 
-    for node in $nodes; do
-        ${WGET_DIR} -q -P "${DESTINATION_DIR}" "${current_server}${scenario}${node}" &
+function download_service_logs {
+    sources=()
+    for file in "$@"; do
+        sources+=("root@services-dev.raiden.network:/home/services/${file}")
     done
 
-    latest_sp_log=$(${CURL_COMMAND} "${current_server}${scenario}" | grep '^<a ' | cut -d\" -f2 | grep ".log.gz$" | sort | tail -n 1)
```

Comment:
Because scenarios can be moved around (for load distribution), a server may have logs for an older scenario which are not updated anymore. Because of this we can't use the nginx listing as a reliable way of getting the latest logs, we have to list the scenarios that are run on the current server and only download those.